.github/workflows/kubeapps-general.yaml

```diff
@@ -592,20 +592,6 @@ jobs:
             docker push $prod_image
           done
 
-  srp_report:
-    needs:
-      - setup
-      - push_images
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/actions/srp-source-provenance
-        with:
-          SRP_CLI_VERSION: ${{env.SRP_CLI_VERSION}}
-          SRP_CLIENT_ID: ${{secrets.SRP_CLIENT_ID}}
-          SRP_CLIENT_SECRET: ${{secrets.SRP_CLIENT_SECRET}}
```

Comment:
Maybe we should delete these vars  (`SRP_CLI_VERSION`, `SRP_CLIENT_ID` and `SRP_CLIENT_SECRET`)  from other places as well?

Reply:
+1